src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -821,16 +821,18 @@ private void buildMiscellaneousCaseNotesEvent(EventHistory.EventHistoryBuilder b
                          .dateReceived(caseNote.getCreatedOn())
                          .eventDetailsText(left((format(
                              "case note added: %s",
-                             caseNote.getNote().replaceAll("\\s+", " ")
+                             caseNote.getNote() != null
```

Comment:
We can use apache common replaceAll which don't require null check

Reply:
its existing code